runelite-client/src/main/java/net/runelite/client/plugins/corp/CorpPlugin.java

```diff
@@ -140,6 +150,22 @@ public void onNpcSpawned(NpcSpawned npcSpawned)
 				break;
 			case NpcID.DARK_ENERGY_CORE:
 				core = npc;
+				if (config.notifyDarkCore() && !coreSpawned)
+				{
+					notifier.notify("Dark core has spawned!");
+
+					String message = new ChatMessageBuilder()
+							.append(ChatColorType.HIGHLIGHT)
+							.append("Dark core has spawned!")
+							.build();
+
+					chatMessageManager.queue(QueuedMessage.builder()
```

Comment:
The notifier already has support for sending notifs to the chatbox, so this isn't necessary.